src/Main.css

```
@@ -0,0 +1,341 @@
+    color: var(--failure);
+}
+
+.outline-badge-primary{
+    border: 2px solid var(--primary);
+    color: var(--primary);
+}
+
+.outline-badge-secondary{
+    border: 2px solid var(--secondary);
+    color: var(--secondary);
+}
+
+.button-solid{
+    color: var(--white);
+    outline: none;
+    border: 0px;
+    margin: 1rem;
+    padding: 0.5rem;
+    font-size: 1rem;
+}
+
+.button-primary{
+    background-color: var(--primary);
+}
+
+.button-secondary{
+    background-color: var(--secondary);
+}
+
+.button-success{
+    background-color: var(--success-mild);
+}
+
+.button-success:hover{
+    background-color: #209892;
+}
+
+.button-warning{
+    background-color: #b71212;
+}
+
+.button-warning:hover{
+    background-color: #ff4848;
+}   
+
+.button-outline{
+    outline: none;
+    margin: 1rem;
+    padding: 0.5rem;
+    font-size: 1rem;
+}
+
+
+.button-primary{
+    color: var(--primary);
+    border: 2px solid var(--primary);
+}
+
+.button-primary:hover{
+    color: var(--white);
+    background-color: var(--primary);
+}
+
+.button-secondary{
+    color:var(--secondary);
+    border: 2px solid var(--secondary);
+}
+
+.button-secondary:hover{
+    color: var(--white);
+    background-color: var(--secondary);
+}
+
+.button-success{
+    color: var(--success-mild);
+    border:2px solid var(--success-mild)
+}
+
+.button-success:hover{
+    color: var(--white);
+    background-color: #209892;
+}
+
+.button-warning{
+    color: var(--failure);
+    border: 2px solid salmon; 
+}
+
+.button-warning:hover{
+    color: var(--white);
+    background-color: #ff4848;
+}
+
+.card-container{
+    margin: 1rem;
+    max-width: 17rem;
+    min-height: 20rem;
+    display: inline-block;
+    box-sizing: border-box;
+    padding:0.5rem;
+    border-bottom-right-radius: 1rem;
+    box-shadow: 7px 9px 13px 5px var(--neutral-mild);
+}
+
+.card-image{
+    width: 250px;
+    height: 250px;
+}
+
+.typography{
+    box-sizing: border-box;
+    padding: 0.5rem;
+}
+
+.typography h4{
+    margin: 0px;
+    padding-bottom: 1rem;
+}
+
+.typography p{
+    width: 15rem;
+    margin: 0px;
+}
+
+.image-outline{
+    width: 15rem;
+    height: 15rem;
+    border-radius: 0.5rem;
+    display: inline-block;
+    margin: 1rem;
+}
+
+.image{
+    width: 15rem;
+    height: 15rem;
+    border-radius: 1rem;
+    margin-left: -1rem;
+    margin-top: -1rem;
+}
+
+.image-outline-primary{
+    border: 2px solid var(--primary);
+}
+
+.image-outline-secondary{
+    border: 2px solid var(--secondary);
+}
+
+.image-responsive{
+    width: 100%;
+    height: 40vh;
+}
+
+.textbox{
+    padding: 0.5rem;
+    outline: 0px;
+    border:0px;
+    letter-spacing: 1px;
+    background-color: var(--neutral-mild);
+}
+
+.textbox:focus{
+    border: 2px solid var(--black);
+}
+
+.textarea{
+    padding: 0.5rem;
+    outline: 0px;
+    border:0px;
+    width: 25rem;
+    height: 10rem;
+    resize: none;
+    letter-spacing: 1px;
+    background-color: var(--neutral-mild);
+    outline: 0px;
+}
```

```suggestion
.textarea{
    padding: 0.5rem;
    outline: 0;
    border:0;
    width: 25rem;
    height: 10rem;
    resize: none;
    letter-spacing: 1px;
    background-color: var(--neutral-mild);
 
}
```
Same refactoring as earlier.